File: JavaScriptBasics/WhileLoop/Lab/Password.js
function password(input){

    let name = String(input[0]);
    let password = String(input[1]);
    let isdone = false;
    let index = 2;
    while(!isdone){
      
        let passwordTry = String(input[index]);
        if(password===passwordTry){
            isdone = true;
            console.log(`Welcome ${name}!`);
        }
        index++;
    }

}

File: JavaBasics/WhileLoops/Lab/Password.java
package EntryModule.WhileLoops.Lab;

import java.util.Scanner;
public class Password {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String name = scanner.nextLine();
        String password = scanner.nextLine();

        String triy = scanner.nextLine();
        if(triy.equals(password)){
            System.out.printf("Welcome %s", name);System.out.print("!");
        }

        while(!triy.equals(password)){
            triy = scanner.nextLine();
            if(triy.equals(password)){
                System.out.printf("Welcome %s", name); System.out.print("!");

            }
        }




    }
}
